.github/workflows/ci.yml

```diff
@@ -85,11 +85,7 @@ jobs:
     timeout-minutes: 15
     needs: [project]
 
-    strategy:
-      matrix:
-        os: [ubuntu-18.04, ubuntu-22.04]
```

Comment:
Both 20.04 and 22.04 images use the same Linux kernel version, 5.15.0-1022-azure, so it didn't make sense to test both.